build/js/live-editor.tooltips.js

```diff
@@ -992,10 +992,10 @@ window.ScratchpadAutosuggestData = {
             description: i18n._("Calculates a color or colors between two color at a specific increment. The amount parameter is the amount to interpolate between the two values where 0.0 equal to the first point, 0.1 is very near the first point, 0.5 is half-way in between, etc."),
             params: [i18n._("c1: Interpolate from this color"), i18n._("c2: Interpolate to this color")]
         }, {
-            name: "colorMode(MODE)",
+            name: "colorMode(MODE, *range)",
```

Comment:
Can you change this to `"colorMode(MODE, *maxValue)"`?  I'm not sure why they use the word `range` in the official docs, but it seems confusing.